copy/single.go

```diff
@@ -248,6 +267,7 @@ func (c *copier) copySingleImage(ctx context.Context, unparsedImage *image.Unpar
 				manifest:         attemptedManifest,
 				manifestMIMEType: manifestMIMEType,
 				manifestDigest:   attemptedManifestDigest,
+				compressionAlgorithms: compressionAlgos,
```

Comment:
I’d probably add it to `wipResult` in one place, either after the loop, or at the very end when forming the final value, to reduce duplication / risk of drift.